libs/domain/picking/src/components/customer-note/src/customer-note.def.ts

```diff
@@ -0,0 +1,7 @@
+import { componentDef } from '@spryker-oryx/core';
+
+export const customerNoteComponent = componentDef({
+  name: 'oryx-customer-note',
+  impl: () =>
+    import('./customer-note.component').then((m) => m.CustomerNoteComponent),
+});
```

Comment:
Should we have a schema here like we have in picking-lists and picking-list-item componemts?

Reply:
Schema is required for options, I dunno what options should we have here?